src/index.html

```diff
@@ -17,6 +17,78 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo-moyo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="is-active nav__link"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav__item">
```